pkg/controller/performanceprofile/components/tuned/tuned_test.go

```diff
@@ -176,8 +176,83 @@ var _ = Describe("Tuned", func() {
 					channelsRegex := regexp.MustCompile(`\s*channels=combined ` + strconv.Itoa(reserveCPUcount) + `\s*`)
 					Expect(channelsRegex.MatchString(manifest)).To(BeTrue())
 				})
-				It("should set specific devices with reserved CPUs count", func() {
-					netDeviceName := "enp0s4"
+				It("should set by interface name with reserved CPUs count", func() {
+					netDeviceName := "eth*"
+					profile.Spec.Net = &performancev2.Net{
+						UserLevelNetworking: pointer.BoolPtr(true),
+						Devices: []performancev2.Device{
+							{
+								InterfaceName: &netDeviceName,
+							},
+						}}
+					manifest := getTunedManifest(profile)
+					reservedSet, err := cpuset.Parse(string(*profile.Spec.CPU.Reserved))
+					Expect(err).ToNot(HaveOccurred())
+					reserveCPUcount := reservedSet.Size()
+					//regex field should be: devices_udev_regex=r'^INTERFACE_NAME=eth.*''
+					devicesUdevRegex := "r'\\^INTERFACE_NAME=" + strings.Replace(netDeviceName, "*", "\\.\\*", -1) + "'"
```

Comment:
this is a bit harder to read than it could. Tests should be optimized for clarity, so I'd just write at the beginning of the block
```
netDeviceName := "eth0*"
devicesUdevRegex := "r\\'^INTERFACE_NAME=eth.*'
```